AllReadyApp/Web-App/AllReady/ViewModels/Home/Index.cs

```diff
@@ -1,11 +1,10 @@
 ﻿using System.Collections.Generic;
-using AllReady.ViewModels.Campaign;
 
 namespace AllReady.ViewModels.Home
 {
-    public class HomePageViewModel
+    public class Index
```

Comment:
@mgmccarthy Sounds good. I too much prefer smaller, very specific view models. Generic ones just become a pain and bloated with unused properties. 100% behind that refactor.